src/createQueryGroup.ts

```diff
@@ -224,7 +237,7 @@ export function createQueryGroup<
     hasTag,
     execute,
     get active() {
-      return subscriptions.size > 0
+      return isActive()
```

Comment:
We probably don't even need to expose this getter at all if we're calling this new onDispose callback.